src/index.html

```diff
@@ -4,9 +4,270 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__link">
+        <img class="header__logo" src="images/logo.png"
+          alt="moyo logo">
+      </a>
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple"
+            class="nav__link is-active"
+            >
+            Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#Samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Laptops" class="nav__link"
+            data-qa="nav-hover"
+            >
+            Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#Gadgets" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Tablets" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Photo" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="container">
+
+      <div class="product-card" data-qa="card">
+        <img src="./images/imac.jpeg" alt="Apple iMac Retina 5k Monoblock"
+        class="product-card__image"
+        >
+        <h2 class="product-card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+        <p class="product-card__code">Product code: 195434</p>
+        <div class="product-card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="product-card__reviews-p">Reviews: 5</p>
+        </div>
+        <div class="product-card__price">
+          <p class="product-card__price-word">Price:</p>
+          <p class="product-card__price-num">$2,199</p>
```

Comment:
Additionally, avoid abbreviations, you can call it:

```suggestion
          <p class="product-card__price-word">Price:</p>
          <p class="product-card__price-value">$2,199</p>
```